components/script/dom/event.rs

```diff
@@ -330,8 +330,8 @@ impl Runnable for EventRunnable {
 
     fn handler(self: Box<EventRunnable>) {
         let target = self.target.root();
-        let bubbles = self.bubbles.clone();
-        let cancelable = self.cancelable.clone();
+        let bubbles = self.bubbles;
+        let cancelable = self.cancelable;
```

Comment:
I moved them out because the borrow check was complaining that `self` was moved because of `self.name` in the next line.